### PR TITLE
impr(quote): Added three english and three hindi quotes

### PR DESCRIPTION
**Pull request by himanshubalani**
### Description

I added Three English Quotes and Three Hindi quotes in quotes.json file.

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38714,6 +38714,24 @@
...
       "id": 6990
+    },
+    {
+      "text": "For myself I am an optimist. It does not seem to be much use being anything else.",
+      "source": "Winston Churchill",
+      "length": 81,
+      "id": 6991
+    },
+    {
+      "text":"Depend upon it there comes a time when for every addition of knowledge you forget something that you knew before. It is of highest importance,therefore, not to have useless facts elbowing out the useful ones.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38725]: missing space after comma

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38714,6 +38714,24 @@
...
+      "source": "The Adventures of Sherlock Holmes",
+      "length": 208,
+      "id": 6992
+    },
+    {
+      "text": "How we spend our days is, of course, how we spend our lives.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38731]: i think  remove this one

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -38714,6 +38714,24 @@
       "source": "Simon Sinek",
       "length": 314,
       "id": 6990
+    },
+    {
+      "text": "For myself I am an optimist. It does not seem to be much use being anything else.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38719]: add comma after myself?

File: frontend/static/quotes/hindi.json [hunk 4]
```diff
@@ -210,6 +210,24 @@
...
+    {
+      "text" : "काल करे सो आज कर, आज करे सो अब । पल में परलय होएगी, बहुरि करेगा कब ।",
+      "source":"कबीर दास",
+      "length": 68,
+      "id" : 36
+    },
+    {
+      "text":"चलती चक्की देख के, दिया कबीरा रोये । दो पाटन के बीच में, साबुत बचा न कोए ।",
+      "source":"कबीर दास",
+      "length": 74,
```
Comment (Miodec) [inline frontend/static/quotes/hindi.json:229]: what do these translate to?

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/quotes/hindi.json (hunk 4 above)
Comment (himanshubalani) [reply frontend/static/quotes/hindi.json:229]: 1. "सब धरती काजग करू, लेखनी सब वनराज । सात समुद्र की मसि करूँ, गुरु गुण लिखा न जाए ।" translated is "
"Let me write on the whole earth,
let the pen be the forest king.
Let me write on the seven seas,
Let the Guru's teachings not be written down."

2. "काल करे सो आज कर, आज करे सो अब । पल में परलय होएगी, बहुरि करेगा कब ।" translated is 
"Do tomorrow's work today, today’s work now; If the moment is lost, how will the work be done?"

3. "चलती चक्की देख के, दिया कबीरा रोये । दो पाटन के बीच में, साबुत बचा न कोए ।" translated is 
"Seeing the moving mill, Saint Kabira cried. In the midst of dumping, no one can save the whole."

They don't make much sense in English, But these are short poems by poet "कबीर दास" (Kabir Das) in Hindi taught in most schools.
I figure if something known is seen by the user, they're likely to practice better

File: frontend/static/quotes/english.json (hunk 1 above)
Comment (himanshubalani) [reply frontend/static/quotes/english.json:38725]: Done

File: frontend/static/quotes/english.json (hunk 3 above)
Comment (himanshubalani) [reply frontend/static/quotes/english.json:38719]: Added Comma

File: frontend/static/quotes/english.json (hunk 2 above)
Comment (himanshubalani) [reply frontend/static/quotes/english.json:38731]: Removed

**Review by Miodec — APPROVED** [review]

**Pull request merged**